src/Hooks/useNewDApps/useNewDApps.ts

```diff
@@ -0,0 +1,37 @@
+import { useMemo } from "react"
+import { useVeBetterDaoDapps } from "~Hooks/useFetchFeaturedDApps"
+import { useAppSelector, selectFeaturedDapps } from "~Storage/Redux"
+import moment from "moment"
+
+export const useNewDApps = () => {
+    const dapps = useAppSelector(selectFeaturedDapps)
+    const { data: veBetterDaoDapps, isLoading } = useVeBetterDaoDapps()
+
+    const newDapps = useMemo(() => {
+        const threeMonthsAgo = moment().subtract(3, "months")
+
+        // Sort from newest to oldest
+        const filteredDapps = dapps
+            .filter(dapp =>
+                veBetterDaoDapps?.find(
+                    vbdDapp =>
+                        dapp.veBetterDaoId === vbdDapp.id &&
+                        moment(dapp.createAt).isAfter(threeMonthsAgo) &&
+                        vbdDapp.appAvailableForAllocationVoting, // Filter out dapps that aren't endorsed
+                ),
+            )
+            .sort((a, b) => b.createAt - a.createAt)
+
+        // If there are no new dapps, return the 10 newest dapps
+        if (filteredDapps.length === 0) {
+            return dapps
+                .filter(dapp => veBetterDaoDapps?.find(vbdDapp => vbdDapp.id === dapp.veBetterDaoId))
+                .sort((a, b) => b.createAt - a.createAt)
+                .slice(0, 10)
+        }
+
+        return filteredDapps.slice(0, 15)
+    }, [dapps, veBetterDaoDapps])
```

Comment:
Should we just search for VBD dapps? Because some dapps don't have an app on app-hub.
We could map them to a DiscoveryDapp if needed for type-safety